src/pfs/pfs.proto

```diff
@@ -631,6 +644,8 @@ service API {
   rpc SquashCommitSet(SquashCommitSetRequest) returns (google.protobuf.Empty) {}
   // DropCommitSet drops the commits of a CommitSet and all data included in the commits.
   rpc DropCommitSet(DropCommitSetRequest) returns (google.protobuf.Empty) {}
+  // SearchForFileInBranch searches for commits that reference a supplied file being modified in a branch.
+  rpc SearchForFileInBranch(SearchForFileInBranchRequest) returns (SearchForFileInBranchResponse) {}
```

Comment:
Are we not returning a stream here anymore? TBH the response object looks good and we are not really using streams correctly in console ATM (we read until stream is closed in backend and return result 😆).

Reply:
That's correct. Its a non-streamed response

Reply:
works for me